README.md

```diff
@@ -75,12 +67,12 @@ This way, NetTracer's going to start in logging mode - all the information about
 
 Note that you need the following capabilities in order to run NetTracer:
 
-- _CAP\_DAC\_READ_SEARCH__
```

Comment:
Why set of required capailities have changed. e.g. _CAP\_DAC\_READ_SEARCH__ and entry for - _CAP\_SYS\_ADMIN_ ?

Reply:
I think an old version of master might have been set as target branch...